### DISCUSSION TITLE
Solicitação de subdomínio para pythonnordeste.org

### DISCUSSION THREAD

**Discussion by itepifanio** [apyb/comunidade#129] [Geral]
Hoje já temos os subdomínios por ano `2024.pythonnordeste.org`, gostaríamos de ter também um `blog.pythonnordeste.org`

**Comment by cecivieira** [comment]
@itepifanio o que tu acha de fazer as contibuições no https://blog.pythonbrasil.org.br/?
Acredito que contribuir com um lugar centralizado para compartilhar conhecimento seja mais interessante

**Reply by itepifanio** [reply]
@cecivieira é uma ideia a ser considerada, qual o repositório do blog? Não consegui achar

Pra funcionar acho que teria que adicionar uma funcionalidade de separação por tags, permitindo que os eventos regionais consigam filtrar seus posts. 

Eu pensei em ser isolado porque dá mais liberdade de customização, a Python Sul por exemplo fez [o blog deles](https://sul.python.org.br/blog/) com a carinha do evento.

**Reply by cecivieira** [reply]
o repositório do blog é esse: https://github.com/pythonbrasil/pybr-blog

sobre customização: todo ano mudamos de identidade visual, então acho que rapidamente ficaria obsoleto, pois não é uma atividade essencial para que o evento aconteça. As organizações costumam ser sobrecarregadas. Além disso, acho que centralizar ajuda a encontrar experiências de organização de evento em um só lugar.

Fica a sugestão para @talitarossari python sul também centralizar.

A ideia de colocar tags é bem boa!!

**Reply by itepifanio** [reply]
@cecivieira abri [uma issue](https://github.com/pythonbrasil/pybr-blog/issues/16) no blog para adicionar essa funcionalidade de tags, uma vez que ela seja adicionada acho que fica fácil de receber essas colaborações. Vou terminar de fechar os pagamentos da PyNE, se até lá ninguém tiver pego essa issue eu consigo implementar.

> então acho que rapidamente ficaria obsoleto

Sim, tem razão. Acho que o que me incomoda mesmo é que o blog da PyBR é simples demais. Adicionar pelo menos uma imagem junto a listagem do título já ajudaria a chamar mais atenção.

De toda forma, estou de acordo em centralizar os posts, podemos seguir dessa forma.

**Reply by naanadr** [reply]
Tbm estou contigo @itepifanio, meu incomodo também é por simples de mais o template 😂

O que tu acha da gente parear um pouco e dar uma repaginada no blog? Além das tags, podemos colocar uma barrinha de pesquisa, entre outras coisas.

**Reply by itepifanio** [reply]
Claro @naanadr, podemos parear sim. Inclusive eu abri um PR no site da PyBR com [o CI automatizado para deploy](https://github.com/pythonbrasil/pybr2024-site/pull/17), isso pode ser útil pro blog também.
